<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="SuperTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>631</width>
      <height>411</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loadBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Load ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rescanBtn">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveasBtn">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>540</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save as ...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Line :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="columnNo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Column :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="byteIdx">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ByteIdx :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="charIdx">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>110</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CharIdx :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qtcharIdx">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Qt CharIdx :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="hoveredLine">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>631</width>
      <height>81</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="globalFileEOL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>File Global EOL Setting</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="f_eol_unix">
        <property name="text">
         <string>Unix (LF \n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="f_eol_win">
        <property name="text">
         <string>Win (CR-LF \r\n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="f_eol_mac">
        <property name="text">
         <string>Mac (CR \r)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="f_eol_hybrid">
        <property name="text">
         <string>HyBrid</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="lineEOL">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>300</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Line EOL Setting</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="eol_unix">
        <property name="text">
         <string>Unix (LF \n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="eol_win">
        <property name="text">
         <string>Win (CR-LF \r\n)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="eol_mac">
        <property name="text">
         <string>Mac (CR \r)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="JumpBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>206</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Jump</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>144</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="byteChkBox">
        <property name="text">
         <string>Byte</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="qtCursorChkBox">
        <property name="text">
         <string>QtCursor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>41</width>
       <height>9</height>
      </rect>
     </property>
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="positionEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="jmpBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>170</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Unicode :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="unicodeValue">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>170</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="forceRead">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>540</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Read TextBox</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SuperTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>SuperTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
